### PR TITLE
Caching By CacheSubscriber -> CacheStorage -> ApcCache

### PR DESCRIPTION
**Pull request by ViktorParakhonia**

### CODE REVIEW

File: src/AppShed/Extensions/SpreadsheetMapsBundle/Controller/MapsController.php [hunk 1]
```diff
@@ -89,48 +89,53 @@ public function documentAction(Request $request)
...
                             $innerScreen = new Screen($value);
-                            $link = new Link($value);
-                            $screen->addChild($link);
                             $index = false;
-                            $link->setScreenLink($innerScreen);
                         } else {
                             if (!empty($value)) {
-                                $map = false;
-                                if ($name == $address ) {
+                                if ($name == strtolower($address) ) {
```
Comment (mcfedr) [inline src/AppShed/Extensions/SpreadsheetMapsBundle/Controller/MapsController.php]: why are you not settings `$address` to `strtolower($doc->getAddress())` at the beginning, so you don't do it in the loop.

Also, what if `$name` is not all lowercase?

Comment (ViktorParakhonia) [inline src/AppShed/Extensions/SpreadsheetMapsBundle/Controller/MapsController.php]: $name is always not capitalized

чт, 21 мая 2015, 15:43, Fred Cox notifications@github.com:

Comment (ViktorParakhonia) [inline src/AppShed/Extensions/SpreadsheetMapsBundle/Controller/MapsController.php]: Lowercase..

чт, 21 мая 2015, 15:48, Віктор Парахоня stelsvitya@gmail.com:

Comment (mcfedr) [inline src/AppShed/Extensions/SpreadsheetMapsBundle/Controller/MapsController.php]: Ok, but you can more the `strtolower` of `$address`

Comment (ViktorParakhonia) [inline src/AppShed/Extensions/SpreadsheetMapsBundle/Controller/MapsController.php]: ok

**Pull request merged**